src/dlsnode/main.d

```diff
@@ -24,7 +24,7 @@ import ocean.transition;
 
 import ocean.io.select.client.SelectEvent;
 import ocean.util.app.DaemonApp;
-import ocean.util.log.Log;
+static import ocean.util.log.Logger;
```

Comment:
Why going static here? (Just interested.)

Reply:
`Log.stats()` seems that it uses the deprecated `Log` import leak through some of the imports in class. I can add `FIXME: remove in ocean v4.x.x` if you like?

Reply:
Sounds good.